src/xdocs/config_filters.xml

```diff
@@ -991,6 +991,216 @@ public class InputTest {
+        </p>
+      </subsection>
+      <subsection name="Properties" id="SuppressionXpathSingleFilter_Properties">
+        <table>
+            <tr>
+                <th>name</th>
+                <th>description</th>
+                <th>type</th>
+                <th>default value</th>
+                <th>since</th>
+            </tr>
+            <tr>
+                <td>files</td>
+                <td>a Regular Expression matched against the file name associated
+                    with an audit event. It is optional.</td>
+                <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>checks</td>
+                <td>a Regular Expression matched against the name of the check
+                    associated with an audit event. Optional as long as id or message
+                    is specified.</td>
+                <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>message</td>
+                <td>a Regular Expression matched against the message of the check associated
+                    with an audit event. Optional as long as checks or id is specified.</td>
+                <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>id</td>
+                <td>a string matched against the ID of the check associated with an audit event.
+                    Optional as long as checks or message is specified.</td>
+                <td><a href="property_types.html#string">String</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>query</td>
+                <td>a string xpath query. It is optional.</td>
+                <td><a href="property_types.html#string">String</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+        </table>
+      </subsection>
+      <subsection name="Examples" id="SuppressionXpathSingleFilter_Examples">
+        <p>
+          To configure to reject CyclomaticComplexity errors
+          for all methods with name sayHelloWorld
+          inside FileOne ane FileTwo classes:
+        </p>
+        <pre>
+&lt;module name="SuppressionXpathSingleFilter"&gt;
+  &lt;property name="files" value="FileOne.java, FileTwo.java"/&gt;
+  &lt;property name="checks" value="CyclomaticComplexity"/&gt;
+  &lt;property name="query" value="//METHOD_DEF[@text='sayHelloWorld']"/&gt;
+&lt;/module&gt;
+        </pre>
+        <p>
+          To suppress checks for package definitions:
```

Comment:
I would be slightly more specific in these situations.
`suppress checks for package definitions` => `suppress checks for all package definitions`.

Since there are no files specified, it will suppress all package nodes in all files.

Reply:
done.